WeakAuras/Prototypes.lua

```diff
@@ -6161,6 +6161,16 @@ Private.event_prototypes = {
         store = true,
         conditionType = "string"
       },
+      {
+        name = "itemId",
+        display = L["ItemId"],
+        hidden = true,
+        init = "item",
+        test = "true",
+        store = true,
+        conditionType = "number",
+        operator_types = "only_equal",
+      },
```

Comment:
Most item triggers use property `hasItemID = true,`  that is only use for tooltips, i wonder if that property can be use for refactor more of these trigger filters / conditions